pandas/core/generic.py

```diff
@@ -546,6 +546,52 @@ def swapaxes(self, axis1, axis2, copy=True):
     def pop(self, item):
         """
         Return item and drop from frame. Raise KeyError if not found.
+
+        Parameters
+        ----------
+        item : str
+            column label to be popped
+
+        Returns
+        -------
+        popped : Series
+
+        Examples
+        --------
+        >>> import numpy as np
+        >>> import pandas as pd
```

Comment:
In general, we don't include the standard imports (so those two) in the docstrings. Also in the tutorial docs we assume people have done these imports to not always have to repeat them